### PR TITLE
Galactic Radio Telescope

### PR DESCRIPTION
**Pull request by hexylena**
Howdy all,

I'd like to re-introduce this project:

The Galactic Radio Telescope (GRT) has had some progress today, and it's gotten to a point where it can be used to start collection of data, while looking towards analysing that data quite a bit further down the line.
## What?

The GRT aims to collect job run information from Galaxy servers that opt into the process.

Galaxy Admins can sign in to the GRT, register their Galaxy servers, and get back API keys per-Galaxy instance. Then they set up a cron job (preferably weekly, but we may explore options here later), and submit job data to the GRT.
## Why?

https://oc.hx42.org/grt/ the about page should cover most questions.
## GRT Repo

https://github.com/erasche/galactic-radio-telescope
## Demo Instance

I've deployed a demo instance here:
https://oc.hx42.org/grt/

And I've posted all of my org's Galaxy's data to it.

utvalg_837

(Warning, accessing things like the "CPT Phage Galaxy" information page in the Galaxy list will be SLOW. It's running off of sqlite, with a large number of uncached queries. If this PR gets accepted, I'll optimize it a lot)

### CODE REVIEW

**Comment by martenson** [comment]
It is very nice Eric! I see great potential. :koala:

**Comment by natefoo** [comment]
I want this.

**Comment by afgane** [comment]
Potential platform for analyzing the collected data: https://www.tacc.utexas.edu/research-development/tacc-projects/karnak

**Comment by hexylena** [comment]
After talking to people, this has been deemed a good idea / useful. 

As such I'll be rewriting parts (the whole) of the backend to be a bit more efficient and cleanly architected.

That said, the script portion of it is mostly done (minus sending all tool params) which we'll add shortly

File: scripts/grt.py [hunk 1]
```diff
@@ -7,6 +7,8 @@
 import argparse
 import sqlalchemy as sa
 import yaml
+import copy
```
Comment (hexylena) [inline scripts/grt.py]: unused import

**Comment by jmchilton** [comment]
:+1: if changes in https://github.com/erasche/galaxy/pull/16 are acceptable and accepted. Thanks for working on this @erasche - this is a great start!

**Comment by hexylena** [comment]
Thanks @ericenns, awesome work :) It's definitely a start @jmchilton, hopefully we can start from here and improve over time. I was really waiting on this to be accepted before putting more time into the backend, so if you're +1ing it... :)

**Pull request merged**